library/ZendService/Twitter/Twitter.php

```diff
@@ -75,6 +75,13 @@ class Twitter
     protected $oauthConsumer = null;
 
     /**
+     * Response
+     *
+     * @var Twitter\Response
+     */
+    protected $twitterResponse = null;
```

Comment:
`private`, unless there are strong reasons for making this protected. The `= null` is redundant`